cluster/images/obs-csi-plugin/entrypoint.sh

```diff
@@ -39,16 +30,24 @@ else
     exit
 fi
 
-echo "Starting deploy obs csi-plugin...."
+echo "Starting install obsfs...."
 mkdir -p /dev/csi-tool/
 tar -zxvf /root/$fileName.tar.gz -C /dev/csi-tool/
 $HOST_CMD cp -r /dev/csi-tool/$fileName/. ./
 $HOST_CMD bash install_obsfs.sh
 
-echo "Starting run obs socket server...."
+echo "Starting install obs socket-server...."
+rm -rf /dev/csi-tool/socket-server
+rm -rf /dev/csi-tool/connector.sock
 cp /bin/socket-server /dev/csi-tool/socket-server
 chmod 755 /dev/csi-tool/socket-server
-$HOST_CMD /dev/csi-tool/socket-server >log.out 2>&1 &
+
+echo "Starting install obs socket service...."
+cp /bin/socket-server.service /dev/csi-tool/socket-server.service
+$HOST_CMD cp /dev/csi-tool/socket-server.service /lib/systemd/system/socket-server.service
```

Comment:
Do we need to delete the file exists?

Reply:
I don't think it's necessary to delete the exist service file. The new files would cover the old files with "CP" action